tutorials/tour/_posts/2017-02-13-classes.md

```diff
@@ -9,46 +9,103 @@ categories: tour
 num: 4
 next-page: traits
 previous-page: unified-types
+topics: classes
+assumed-knowledge: no-return-keyword, type-declaration-syntax, string-interpolation, procedures
 ---
 
-Classes in Scala are static templates that can be instantiated into many objects at runtime.
-Here is a class definition which defines a class `Point`:
+Classes in Scala are blueprints for creating objects. They can contain methods,
+values, and variables, which are collectively called _members_.
```

Comment:
They can also contain types, objects, traits, and classes which are also members.